blob/driver/driver.go

```diff
@@ -306,6 +306,8 @@ type Bucket interface {
 type SignedURLOptions struct {
 	// Expiry sets how long the returned URL is valid for. It is guaranteed to be > 0.
 	Expiry time.Duration
+	// Method is the HTTP method that can be used on the URL; one of "GET", "PUT", or "DELETE".
```

Comment:
nit: this should clarify to the drivers that only one of the three will be passed in, thus let the driver know that they must handle all of the three.

Reply:
Done.